L10nSnapshotTests/L10nSnapshotTests.swift

```diff
@@ -6,88 +6,54 @@ import XCTest
 
 class L10nSnapshotTests: L10nBaseSnapshotTests {
     func test02DefaultTopSites() {
+        navigator.goto(HomePanelsScreen)
         snapshot("02DefaultTopSites-01")
     }
 
     func test03MenuOnTopSites() {
-        let app = XCUIApplication()
-        app.buttons["TabToolbar.menuButton"].tap()
+        navigator.goto(NewTabScreen)
+        navigator.goto(BrowserTabMenu)
         snapshot("03MenuOnTopSites-01")
-        app.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.25)).tap()
-
-        loadWebPage(url: "about:blank", waitForLoadToFinish: false)
-        sleep(2)
-        app.buttons["TabToolbar.menuButton"].tap()
-        snapshot("10MenuOnWebPage-01")
-        app.otherElements["MenuViewController.menuView"].swipeLeft()
-        snapshot("10MenuOnWebPage-02")
-        app.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.25)).tap()
-
-        app.buttons["URLBarView.tabsButton"].tap()
-        app.buttons["TabTrayController.menuButton"].tap()
-        snapshot("10MenuOnTabsTray-02")
     }
 
-    func test04Foo() {
-        let app = XCUIApplication()
-        loadWebPage(url: "http://wopr.norad.org/~sarentz/fxios/testpages/index.html", waitForOtherElementWithAriaLabel: "body")
-        app.buttons["TabToolbar.menuButton"].tap()
-        snapshot("10MenuOnWebPage-01")
-        app.otherElements["MenuViewController.menuView"].swipeLeft()
-        snapshot("10MenuOnWebPage-02")
+    func scroll(_ table: XCUIElement, eachScreen: (Int) -> ()) {
+        // TODO implement steadily scrolling up the screen
+        // hint: find the last cell that is hittable, and swipe up until it isn't
```

Comment:
this will be great to have for the top level settings screen. I already broke this screen's l10n once by accident.